Assets/Decontainment/UI/CodeEditor/Scripts/Banner.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+using TMPro;
+
+public class Banner : MonoBehaviour
+{
+	[SerializeField]
+	private int ticks = 5;
+
+    private int counter = -1;
+    private Image image;
+    private TextMeshProUGUI text;
+
+    protected Color defaultColor = new Color(0, 0, 0, 0);
+
+    // Start is called before the first frame update
+    void Start()
+    {
+        image = GetComponent<Image>();
+        text = GetComponentInChildren<TextMeshProUGUI>();
+        image.enabled = false;
+        text.enabled = false;
+    }
+
+    // Update is called once per frame
+    void Update()
+    {
+		if (counter >= 0)
+        {
```

Comment:
This bracket should be on the same line as the if statement it corresponds to. This is mentioned in the style guide.